doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
This is making it less consistent, diverging in this file from what is more commonly used in the rest of the docs.